plugins/coverbrowser.koplugin/main.lua

```diff
@@ -265,6 +270,54 @@ function CoverBrowser:addToMainMenu(menu_items)
                         separator = true,
                     },
                 },
+            },
+            {
+                text = _("Collection display mode"),
```

Comment:
May be plural? `Collections display mode`.
Even if for now, we only have a single collection named `Favorites`. So, to avoid confusion, may be `Favorites display mode` until we get more :).
(Then, the only place we would see the word "Collection" is when holding on a file in the Favorites list, which would be ok - if I didn't miss any other.)